### PR TITLE
Setup unit testing

### PR DESCRIPTION
**Pull request by yneves**
Unit tests stays next to the source code like `AlertModal.spec.js` for `AlertModal.js`.

### CODE REVIEW

**Comment by ntgn81** [comment]
Yep, I like them being next to each other, nice job!

**Comment by ntgn81** [comment]
@yneves , looks like we have linting issues, can you take a look?

File: src/test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+var Modal = require('react-bootstrap/lib/Modal');
+var ReactWrapper = require('enzyme').ReactWrapper;
+
+ReactWrapper.prototype.findModal = function() {
+  var wrapper = this.find(Modal);
+  if (wrapper.length && wrapper.node._modal) {
+    var modal = wrapper.node._modal;
+    return new ReactWrapper(modal, modal);
+  }
+  return modal;
```
Comment (ntgn81) [inline src/test.js]: Do you mean to return `wrapper` here instead of `modal`?

Comment (yneves) [inline src/test.js]: yeah, fixed

**Pull request merged**